extensions/kubeapi/workloads/daemonsets/create.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"github.com/rancher/shepherd/clients/rancher"
+	deflabels "github.com/rancher/shepherd/extensions/defaults/labels"
```

Comment:
I would vote to not shorten the name, and just call it defaultLabels or something so that its more clear